src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,99 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
```

Comment:
Don't use Objects, Arrays, or any other util class. Instead of using `Objects.equals` in the `remove(T element)` method, you should use `element == null ? elements[i] == null : element.equals(elements[i])` to check for equality.

Reply:
fix comment